creature/Conflux;Conflux's Skeletons.json

```diff
@@ -0,0 +1,1274 @@
+										"name": "Intelligence (Athletics) DC 10:",
+										"type": "entries",
+										"entries": [
+											"Despite their murderous intent, ultimately skeletons are kind of just people with no muscles. They are easily shoved around, and a single well placed strike with a bone-breaking weapon such as a cosh or hammer is often enough to neutralize one."
+										]
+									},
+									{
+										"name": "Nature DC 20:",
+										"type": "entries",
+										"entries": [
+											"Though the principle is still the subject of some controversy, it is now thought in many academic circles that all skeletons are independant, intelligent beings, living in symbiosis with the creatures they inhabit.",
+											"If nothing else, this helps to explain the mysterious ease with which a necromancer may independantly animate the flesh and bones of a creature."
+										]
+									}
+								]
+							},
+							{
+								"name": "Skeleton Tactics",
+								"type": "entries",
+								"entries": [
+									"Skeletons typically begin combat by targeting the nearest living creature with ranged or melee attacks, but on subsequent turns prioritize as a target anyone who destroyed a skeleton in the last turn. They are easily lured into traps as they cannot easily perceive nonliving things, but usually will not provoke attacks of opportunity with their Hate the Living reaction."
+								]
+							},
+							{
+								"name": "Skeletal Liberator Lore",
+								"type": "entries",
+								"entries": [
+									{
+										"name": "History DC 20:",
+										"type": "entries",
+										"entries": [
+											"There are occasional stories of whole villages disappearing overnight, their bones tearing free leaving piles of loose flesh behind. Rumors are spoken in hushed voices of a spontaneously arisen, free-willed skeleton that can awaken other skeletons."
+										]
+									}
+								]
+							},
+							{
+								"name": "Skeletal Liberator Tactics",
+								"type": "entries",
+								"entries": [
+									"Skeletal Liberators use their Wake the Bones if they have an unconscious target or one they can drag into a hazard, otherwise preferring to use their Fleshstripper Blade. If it has fewer than 35 hit points, it saves its reaction for Dark Martyr."
+								]
+							}
+						]
+					}
+				]
+			}
+		},
+		{
+			"name": "Bonecrusher Skeleton",
+			"source": "ConfluxsSkeletons",
+			"page": 2,
+			"size": [
+				"H"
+			],
+			"type": "undead",
+			"alignment": [
+				"L",
+				"E"
+			],
+			"ac": [
+				{
+					"ac": 14,
+					"from": [
+						"natural armor"
+					]
+				}
+			],
+			"hp": {
+				"average": 126,
+				"formula": "12d12 + 48"
+			},
+			"speed": {
+				"walk": 30
+			},
+			"str": 21,
+			"dex": 9,
+			"con": 18,
+			"int": 6,
+			"wis": 8,
+			"cha": 5,
+			"senses": [
+				"darkvision 60 ft."
+			],
+			"passive": 9,
+			"resist": [
+				{
+					"resist": [
+						"piercing and slashing"
+					],
```

Comment:
The `resist` array expects specific values. In this case you probably want the following:
```json
"resist": [
    "piercing",
    "slashing"
]
```